src/common/guards/throttler.guard.spec.ts

```
@@ -1,38 +1,219 @@
+
+    // Mock Date.now to return controlled timestamps
+    Date.now = jest.fn(() => mockedTime);
+  });
+
+  afterEach(() => {
+    // Restore original Date.now implementation
+    Date.now = originalDateNow;
+  });
+
+  function createMockExecutionContext(
+    ip: string = '127.0.0.1',
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)